flake.nix

```diff
@@ -2,52 +2,31 @@
   description = "soxin: opiniated configs for everyone";
 
   inputs = {
-    nixpkgs.url = "github:NixOS/nixpkgs";
+    deploy-rs.url = github:serokell/deploy-rs;
+    nixpkgs.url = github:NixOS/nixpkgs/release-21.05;
+    nur.url = github:nix-community/NUR;
+    sops-nix.url = github:Mic92/sops-nix;
+    unstable.url = github:NixOS/nixpkgs/nixos-unstable;
+    utils.url = github:gytis-ivaskevicius/flake-utils-plus/v1.1.0;
+
     home-manager = {
       url = "github:nix-community/home-manager";
       inputs.nixpkgs.follows = "nixpkgs";
     };
-    futils.url = "github:numtide/flake-utils";
   };
 
-  outputs = { self, nixpkgs, home-manager, futils } @ inputs:
-    let
-      inherit (nixpkgs) lib;
-      inherit (lib) recursiveUpdate;
-      inherit (futils.lib) eachDefaultSystem;
-
-      multiSystemOutputs = eachDefaultSystem (system:
-        let
-          pkgs = nixpkgs.legacyPackages.${system};
-        in
-        {
-          devShell = pkgs.mkShell {
-            buildInputs = with pkgs; [
-              git
-              nixpkgs-fmt
-              pre-commit
-            ];
-          };
-
-          packages = self.lib.overlaysToPkgs self.overlays pkgs;
```

Comment:
are we sure we want to drop those?

Reply:
No, I did not intend to, just fell through. I added them back.